Applications/Spire/Source/Ui/ColorSelectorButton.cpp

```diff
@@ -44,6 +45,13 @@ bool ColorSelectorButton::eventFilter(QObject* watched, QEvent* event) {
       m_selector_widget->activateWindow();
       m_selector_widget->setFocus();
     }
+  } else if(watched == this) {
+    if(event->type() == QEvent::KeyPress) {
+      auto e = static_cast<QKeyEvent*>(event);
+      if(e->key() == Qt::Key_Down) {
```

Comment:
What is this section used for?

Reply:
The default behavior of the DropDownWindow is to show itself when its parent receives a down key press. The expectation was that most widgets that use a DropDownWindow would have the down key as an activation option. If the down key is reserved for the list I can move the down key event filter to the list and remove this code.

Reply:
Oh I see, okay keep this then.

Reply:
Also if that's the case then feel free to have the down arrow key also open up the color window as well.

Reply:
Should I ask Darryl to update the user flow?

Reply:
Yep, go for it.